examples/apps/aml/web-client/package-lock.json

```diff
@@ -0,0 +1,12808 @@
+{
```

Comment:
Do we need this package-lock.json file as well? I presume we need only package.json

Reply:
package-lock.json is useful to lock dependencies versions and avoid errors in the future